cli/structure/index.md

```diff
@@ -6,60 +6,85 @@ permalink: /cli/structure/
+The **Storage API token** to your project is stored in the file `.env.local` under `KBC_STORAGE_API_TOKEN` directive. 
+Currently, it is necessary to use [Master tokens](https://help.keboola.com/management/project/tokens/#master-tokens).
+Your token must be secret, so file `.env.local` is included in `.gitignore` file.
+
+[Manifest - Naming](#naming) defines directories names.
+It is usually not necessary to change this setting. 
+It is guaranteed that each object (branch, config, row) will have its own unique directory, 
+even if the objects have the same name.
+
+
+The following is an example of a default project directory structure. 
+Some files and directories are specific to component type. 
+For example, transformations are represented by native files.
+A more detailed description can be found in the chapters below.
+
+<br>
 
 ```
-├─ .gitignore
-├─ .env.local                        - contains API token
-├─ .env.dist                         - template for .env.local
-├─ .keboola                          - metadata
-   └─ manifest.json
-└─ [branch-name]                     - e.g. test
-   ├─ description.md
-   ├─ meta.json                 
-   └─ [component-type]               - e.g. extractor
-      └─ [component-id]              - e.g. keboola.ex-db-oracle
-         └─ [config-name]            - e.g. oauth-test
-            ├─ config.json           
-            ├─ description.md
-            ├─ meta.json        
-            └─ rows
-               └─ [row-name]         - e.g. prod-fact-table
-                  ├─ config.json     
-                  ├─ description.md
-                  └─ meta.json
-            └─ schedules
-               └─ [schedule-name]
-                  ├─ config.json     
-                  ├─ description.md
-                  └─ meta.json
-            └─ variables
-               └─ values
-                  └─ default
-                     ├─ config.json     - variables default values
-                     ├─ description.md       
-                     └─ meta.json
-               ├─ config.json           - variables configuration
-               ├─ description.md
-               └─ meta.json
-   └─ _shared
-      └─ [component-id]
-         └─ codes
-            └─ [code-name]
-               ├─ code.[ext]            - e.g. code.sql
-               ├─ config.json
-               ├─ description.md
-               └─ meta.json
+🟫 .gitignore                   - excludes ".env.local" from git repository
+🟫 .env.local                   - contains Storage API token
+🟫 .env.dist                    - template for .env.local
+📂 .keboola                     - project metadata directory
+┗ 🟦 manifest.json              - object IDs, naming and other configuration
+📂 [branch-name]                - branch directory, e.g. main
+┣ 🟦 meta.json
+┣ 🟩 description.md
+┣ 📂 _shared                    - shared codes directory
+┃ ┗ 📂 [target-component]       - target, e.g. keboola.python-transfomation
```

Comment:
Bol feedback, ze sa v tom neda orientovat, ... tak som to cele presiel, doplnil a trocha ofarbil:
![image](https://user-images.githubusercontent.com/19371734/154908749-a0025c85-164a-47ba-b74e-e36cb03f58b5.png)